### PR TITLE
No issue: Fix add-on translate() breaking change

### PR DESCRIPTION
**Pull request by Amejia481**
Dependes on https://github.com/mozilla-mobile/android-components/pull/6628

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/customtabs/ExternalAppBrowserFragment.kt [hunk 1]
```diff
@@ -84,8 +84,8 @@ class ExternalAppBrowserFragment : BaseBrowserFragment(), UserInteractionHandler
                         activity,
                         components.core.store,
                         customTabSessionId
-                    ) { exception ->
-                        components.analytics.crashReporter.submitCaughtException(exception)
+                    ) { uri ->
+                        components.analytics.crashReporter.submitCaughtException(Exception("Unknown scheme error $uri"))
```
Comment (Amejia481) [inline app/src/main/java/org/mozilla/fenix/customtabs/ExternalAppBrowserFragment.kt:88]: @rocketsroger is this OK for fixing the breaking change?

Comment (rocketsroger) [reply app/src/main/java/org/mozilla/fenix/customtabs/ExternalAppBrowserFragment.kt:88]: Sure, I'll apply my patch after

Comment (Amejia481) [reply app/src/main/java/org/mozilla/fenix/customtabs/ExternalAppBrowserFragment.kt:88]: 👍

**Comment by ekager** [comment]
I also updated a-s to 57 from 55 in https://github.com/mozilla-mobile/android-components/pull/6298 and think we also need to update `Server.dev is now Server.stage to reflect better the FxA server instance it points to.`

**Comment by Amejia481** [comment]
> I also updated a-s to 57 from 55 in [mozilla-mobile/android-components#6298](https://github.com/mozilla-mobile/android-components/pull/6298) and think we also need to update `Server.dev is now Server.stage to reflect better the FxA server instance it points to.` in `FxaServer`

@ekager would you mind pushing these changes? :)

**Pull request merged**